.github/workflows/reusable-e2e-tests.yml

```diff
@@ -1,155 +1,126 @@
+    inputs:
+      e2eBranch:
+        type: string
+        required: false
+        default: 'eth-rollup-develop'
+      substrateNodeVersion:
+        type: string
+        required: false
+        default: 'eth-rollup-develop'
+      globalVersion:
+        type: string
+        required: false
+        default: '8dd0c81aa88b967334d001d8cae031f8799a189f'
 
 permissions:
-  contents: write
-  id-token: write
-  deployments: write
+  contents: read
+  actions: read
   checks: write
 
-concurrency: 
-  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
-  cancel-in-progress: true
+env:
+  AVS_AGGREGATOR_VERSION: ${{ inputs.globalVersion }}
+  AVS_FINALIZER_VERSION: ${{ inputs.globalVersion }}
+  ROLLUP_SEQUENCER_VERSION: ${{ inputs.globalVersion }}
+  ROLLUP_UPDATER_VERSION: ${{ inputs.globalVersion }}
+  MANGATA_NODE_VERSION: ${{ inputs.substrateNodeVersion }}
 
 jobs:
-  init:
-    name: Set global version
-    if: github.event.action != 'closed'
+  run-e2e-tests:
     runs-on: ubuntu-latest
-    outputs:
-      GLOBAL_VERSION: ${{ steps.set_vars.outputs.GLOBAL_VERSION }}
-      NODE_VERSION: ${{ steps.set_vars.outputs.NODE_VERSION }}
-      AVS_VERSION: ${{ steps.set_vars.outputs.AVS_VERSION }}
     steps:
       - uses: actions/checkout@v4
-
-      - name: Load environment variables
-        uses: xom9ikk/dotenv@v2
         with:
-          path: ./
-          load-mode: strict
-        
-      - name: Set global version
-        id: set_vars
-        run: |
-          echo "GLOBAL_VERSION=${{ github.sha }}" >> $GITHUB_OUTPUT 
-          echo "NODE_VERSION=${{ inputs.nodeVersion || env.MANGATA_NODE_VERSION }}" >> $GITHUB_OUTPUT
-          echo "AVS_VERSION=${{ inputs.avsVersion || '8dd0c81aa88b967334d001d8cae031f8799a189f' }}" >> $GITHUB_OUTPUT
-  
-  #build-and-test:
-    #needs: [init]
-    #name: Build
-    #uses: ./.github/workflows/reusable-build-and-test.yml
-    #secrets: inherit
-    #with:
-      #version: ${{ needs.init.outputs.GLOBAL_VERSION }}
-
-  start-env:
-    runs-on: [ubuntu-latest]
-    needs: [init]
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          submodules: 'recursive'
-
-      #- name: build finalizer
-      #  working-directory: avs-finalizer/
-      #  run: cargo +stable build --release
+          submodules: recursive
 
       - name: Start env
-        run: |
-          export AVS_AGGREGATOR_VERSION=${{ needs.init.outputs.AVS_VERSION }}
-          export AVS_FINALIZER_VERSION=${{ needs.init.outputs.AVS_VERSION }}
-          export MANGATA_NODE_VERSION=${{ needs.init.outputs.NODE_VERSION }}
-          docker compose build sequencer updater
-          docker compose up --wait 
-
-#      - name: Sleep for 2 minutes
-#        run: sleep 120s
-
-      - name: docker ps
-        run: docker ps
-
-      - name: E2E- Get branch name
-        id: branch-name
-        uses: tj-actions/branch-names@v7.0.5
-
-      - name: E2E- Calculate if run e2e feature branch or main.
```

Comment:
hey @tenequm , 
is this done elsewhere? 
The idea for this is  that, 
If this jobs runs on a branch -> find a branch in e2e with the same name, and if it does exist , use it.  else -> develop. 
I guess that you removed it ( which is fine, since you are triggering this job from another job) , but I can not find where this fancy feature is done :upside_down_face: